### PR TITLE
[IMP] mrp_operations_extension: new method for computing value

### PR DESCRIPTION
**Pull request by oihane**
This PR solves one of the errors that are making other PR to fail.

### CODE REVIEW

**Comment by agaldona** [comment]
:+1:

File: mrp_operations_extension/tests/test_mrp_operations_extension.py [hunk 1]
```diff
@@ -156,8 +155,10 @@ def test_confirm_production_without_do_production_flag(self):
             self.production.action_confirm()
 
     def test_confirm_production_operation_extension_case1(self):
+        self.assertEqual(self.production.state, 'draft')
         self.production.signal_workflow('button_confirm')
-        self.assertFalse(self.production.workcenter_lines[0].is_material_ready)
+        self.assertFalse(
+            all(self.production.mapped('workcenter_lines.is_material_ready')))
```
Comment (pedrobaeza) [inline mrp_operations_extension/tests/test_mrp_operations_extension.py]: You should put here `any` instead of `all`

**Comment by oihane** [comment]
Travis error solved in this PR

```
FAIL: test_confirm_production_operation_extension_case1 (openerp.addons.mrp_operations_extension.tests.test_mrp_operations_extension.TestMrpOperationsExtension)
Traceback (most recent call last):
`   File "/home/travis/build/OCA/manufacture/mrp_operations_extension/tests/test_mrp_operations_extension.py", line 163, in test_confirm_production_operation_extension_case1
`     self.assertTrue(line.is_material_ready)
` AssertionError: False is not true
```

**Comment by pedrobaeza** [comment]
I still see the error.

**Comment by oihane** [comment]
Travis is still failing because error in `mrp_production_estimated_cost` take a look to issue https://github.com/OCA/manufacture/issues/124#issuecomment-220369506
:wink:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**